packages/astro/src/internal/swap-document.ts

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line import/no-unresolved
+import { swapFunctions } from 'astro:transitions/client';
```

Comment:
❓ This is always bundled with Astro ? Is there a case where the module will not exists causing our integration to break ?

Reply:
It is indeed bundled with Astro (`astro/virtual-modules/transitions-swap-functions.js`), but is not available in Astro v3